.github/workflows/main.yml

```diff
@@ -0,0 +1,26 @@
+name: Docs to PDF
+# This workflow is triggered on pushes to the master branch.
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  converttopdf:
+    name: Create our PDF
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      # Generate PDF for each folder in /contents
+      - uses: mkrakowitzer/actions-makepdfs@master
```

Comment:
Let's us a fixed version of the action to avoid its breaking in the future.

Reply:
Fixed to mkrakowitzer/actions-makepdfs@v1.2